archaius-core/src/main/java/netflix/archaius/PropertyConversions.java

```diff
@@ -0,0 +1,66 @@
+package netflix.archaius;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+/**
+ * Interface for converting properties between various types.
+ */
+public interface PropertyConversions {
```

Comment:
Why do you think we need this abstraction?  I believe the Decoder is better suited for this separation.  The ObservableProperty is more a proxy to making the correct getXXX on Config which in turn should call the Decoder.